independent-publisher-connectors/Proposal - Github as Datasource/apiProperties.json

```diff
@@ -0,0 +1,25 @@
+{
+  "properties": {
```

Comment:
Can you please modify the apiProperties.json file as per the template below. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json